Makefile

```diff
@@ -24,3 +43,75 @@ cluster/kubeconfig:
 .PHONY: cluster/list
 cluster/list:
 	source $(COMMON_SCRIPT) && ls -ltr "$${MON_BENCHMARKS_ROOT}/clusters"
+
+
+##############################
+## Images
+##############################
+PHONY: image/build/local
+image/build/local:
+	docker build -t $(IMAGE_TAG) .
+
+.PHONY: image/shell/local
+image/shell/local:
+	docker run -it --rm --entrypoint /bin/bash $(IMAGE_TAG)
+
+.PHONY: image/push
+image/push: image/build/local
+	echo 'Assumming logged with `docker login -u "${USER}" quay.io`'
+	docker tag $(IMAGE_TAG) quay.io/${USER}/$(IMAGE_NAME):$(IMAGE_VERSION)
+	docker push $(EXTERNAL_IMAGE_TAG)
+	echo "Asumming project is public at https://quay.io/repository/${USER}/$(IMAGE_NAME)?tab=settings "
+
+
+##############################
+## Benchmarks run
+##############################
+$(LOCAL_ROOT):
+	mkdir -p $(LOCAL_ROOT)
+
+$(E2E_BENCHMARKING_ROOT): $(LOCAL_ROOT)
+	ls $(E2E_BENCHMARKING_ROOT) || ( cd $(LOCAL_ROOT) && git clone https://github.com/cloud-bulldozer/e2e-benchmarking.git && cd e2e-benchmarking && git checkout $(E2E_BENCHMARKING_HASH) )
```

Comment:
Why do we need to checkout a specific hash?

Reply:
This is to depend on a specific version of e2e-benchmarking, otherwise the benchmarks can fail on an untested version of e2e-benchmarking after it upgrades